src/main/scala/beam/sim/vehiclesharing/FixedNonReservingFleetManager.scala

```diff
@@ -83,44 +86,25 @@ private[vehiclesharing] class FixedNonReservingFleetManager(
       sender ! MobilityStatusResponse(nearbyVehicles.take(5).map { vehicle =>
         Token(vehicle.id, self, vehicle.toStreetVehicle)
       })
+      self ! REPVehicleInquiry(personId, whenWhere)
 
     case TryToBoardVehicle(token, who) =>
-      availableVehicles.get(token.id) match {
-        case Some(vehicle) if token.streetVehicle.locationUTM == vehicle.spaceTime =>
-          availableVehicles.remove(token.id)
-          val removed = availableVehiclesIndex.remove(
-            new Envelope(new Coordinate(vehicle.spaceTime.loc.getX, vehicle.spaceTime.loc.getY)),
-            vehicle
-          )
-          if (!removed) {
-            log.error("Didn't find a vehicle in my spatial index, at the location I thought it would be.")
-          }
-          who ! Boarded(vehicle)
-          log.debug("Checked out " + token.id)
-        case _ =>
-          who ! NotAvailable
+      println(s"to board ===> ${token.id}")
```

Comment:
It is preferable to use `logger.debug` and adjust debug level for particular class

Reply:
I have already removed all these println lines